osu.Desktop/Updater/VelopackUpdateManager.cs

```diff
@@ -27,143 +27,117 @@ public partial class VelopackUpdateManager : Game.Updater.UpdateManager
         [Resolved]
         private ILocalUserPlayInfo? localUserInfo { get; set; }
 
-        [Resolved]
-        private OsuConfigManager osuConfigManager { get; set; } = null!;
-
         private bool isInGameplay => localUserInfo?.PlayingState.Value != LocalUserPlayingState.NotPlaying;
 
-        private readonly Bindable<ReleaseStream> releaseStream = new Bindable<ReleaseStream>();
-        private UpdateManager? updateManager;
-        private UpdateInfo? pendingUpdate;
```

Comment:
One of the main points is that this entire class has become stateless with regards to the update.

On `master`, the statefulness means that if an update is found for the "Lazer" stream this class would _refuse_ to check for a subsequent update on the "Tachyon" stream.